src/DependencyInjection/CmfSonataAdminIntegrationExtension.php

```diff
@@ -16,19 +16,46 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\DependencyInjection\Factory\AdminFactoryInterface;
+use Symfony\Cmf\Bundle\SeoBundle\CmfSeoBundle;
 
 /**
  * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ * @author Wouter de Jong <wouter@wouterj.nl>
  */
 class CmfSonataAdminIntegrationExtension extends Extension
 {
+    /**
+     * @var AdminFactoryInterface[]
+     */
+    private $factories;
+
+    /**
+     * @param null|AdminFactoryInterface[] $factories A list of Admin factories
+     */
+    public function __construct(array $factories = null)
+    {
+        if (null !== $factories) {
+            $this->factories = $factories;
+        } else {
+            $bundles = [
+                CmfSeoBundle::class => new Factory\SeoAdminFactory(),
+            ];
+
+            foreach ($bundles as $bundleFqcn => $factory) {
+                if (class_exists($bundleFqcn)) {
```

Comment:
Is it not better to rely on enabled bundles instead of installed ones (i.e. using `kernel.bundles` parameter)?